frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -421,9 +424,24 @@ export const getRunStatusColor = (status: string): StatusMessage => {
 export const truncateName = (name: string, length: number): string =>
   name.length < length ? name : `${name.slice(0, length - 1)}...`;
 
-export const getTaskStatus = (pipelinerun: PipelineRun, pipeline: Pipeline): TaskStatus => {
-  const totalTasks =
-    pipeline && pipeline.spec && pipeline.spec.tasks ? pipeline.spec.tasks.length : 0;
+export const getPipelineFromPipelineRun = (pipelineRun: PipelineRun): Pipeline => {
+  if (!pipelineRun?.status?.pipelineSpec) {
+    return null;
+  }
+  return {
+    apiVersion: PipelineModel.apiVersion,
```

Comment:
```suggestion
    apiVersion: apiVersionForModel(PipelineModel),
```
Awkward, I know... but `apiVersion` in the Model is actually the version, whereas the K8s CR is the "group + version".